identity/identity.go

```diff
@@ -28,6 +29,8 @@ func (a AuthMode) String() string {
 		return "Legacy Authentication"
 	case AuthUserPass:
 		return "Username/password Authentication"
+	case AuthV3UserPass:
+		return "Username/password (Version 3) Authentication"
```

Comment:
Username/password Authentication (Version 3) ?

To minimize confusion, we should perhaps clarify which version the original AuthUserPass is using.